pages/base.py

```diff
@@ -139,6 +147,12 @@ def click_logout_menu_item(self):
             self.find_element(*self._logout_menu_item_locator).click()
             self.wait.until(lambda s: not self.is_logout_menu_item_present)
 
+        def click_groups_menu_item(self):
+            self.click_options()
+            self.find_element(*self._groups_menu_item_locator).click()
+            from pages.groups_page import GroupsPage
```

Comment:
Is there a reason you are importing this locally? Does it raise a circular dependency if it's imported in the top-level?

Reply:
yes, it's causing circular dependency if imported in top level